### ISSUE TITLE
Parameter Units for calculate_mean_radiant_temperature, return value of calc cza integrated

### ISSUE THREAD

**Issue by jarataraj** [ecmwf-projects/thermofeel#24]
The example code [calc_utci.py](https://github.com/ecmwf-projects/thermofeel/blob/e37aaed64235fbbef76f7cb6876f27e052438c46/examples/compute_utci.py#L219) and [utci_wbgt.py](https://github.com/ecmwf-projects/thermofeel/blob/e37aaed64235fbbef76f7cb6876f27e052438c46/examples/utci_wbgt.py#L45) deaccumulate ssrd, ssr, fdir, strd, and strr, which leads me to believe the units for these arguments should be fluxes (W/m^2 or J/sm^2) rather than accumulated radiation as stated in the docstring (J/m^2 or Jm^-2). From what I can tell, the [Di Napoli paper](https://link.springer.com/article/10.1007/s00484-020-01900-5) says the MRT calculation is done with surface *fluxes*, not accumulated radiation.

Additionally, the docstring for [calculate_cos_solar_zenith_angle_integrated](https://github.com/ecmwf-projects/thermofeel/blob/e37aaed64235fbbef76f7cb6876f27e052438c46/thermofeel/thermofeel.py#L222) says: returns average of cosine of the solar zenith angle during interval [degrees], but the example code in [calc_utci.py](https://github.com/ecmwf-projects/thermofeel/blob/e37aaed64235fbbef76f7cb6876f27e052438c46/examples/compute_utci.py#L219) and [utci_wbgt.py](https://github.com/ecmwf-projects/thermofeel/blob/e37aaed64235fbbef76f7cb6876f27e052438c46/examples/utci_wbgt.py#L45) "deaccumulate" the value returned by this function as well, which leads me to believe that calculate_cos_solar_zenith_angle_integrated returns the integral of the cos solar zenith angle over a period over time, rather than the average cza during the interval. I'm not familiar enough with the math used to answer this for myself.

To summarize:
1. Are the units for `ssrd, ssr, fdir, strd, strr` in `calculate_mean_radian_temperature` supposed to be W/m^2, and incorrectly labelled as J/m^2?
2. Does calculate_cos_solar_zenith_angle_integrated return the integral of cza over a period or the average cza over a period?

I am new to this so please correct me if I am wrong. Thank you for your help!!

**Comment by tlmquintino** [comment]
This should have now been fixed in the development branch and will be present in version 1.3.0
